### PR TITLE
Add NewJersey ELSA topology information

### PR DESCRIPTION
**Pull request by mmascher**

### CODE REVIEW

**Comment by mmascher** [comment]
@brianhlin , you mean changing OSG_US_NEWJERSEY_ELSA to whatever we agreed here:

https://opensciencegrid.freshdesk.com/a/tickets/27961

So the site name should probably be TCNJ - ELSA, right?

CCing @jdost321 , who said would run Lauren proposal by Tim/Frank?

**Comment by brianhlin** [comment]
Hey @mmascher thanks for the reminder about that! I'll run this by Frank/TimC later this week after the proposal dust has settled. Is there outside pressure to get this site to show up in the GRACC? If not, I say that we hold off on it until we get a decision from management.

In the meantime, though, it looks like you need to add a `FACILITY.yaml` file

**Comment by mmascher** [comment]
No outside pressure, let's wait! I guess I have to use the `bin/next_facility_id` to get the id for the `FACILITY.yaml` file, right?

**Comment by brianhlin** [comment]
Yup!

**Comment by brianhlin** [comment]
Hey @mmascher, I ran this by management and they agreed with our assessment in the Freshdesk ticket above. So let's use your suggestion and rename the site to `TCNJ - Elsa` then add the `facility.yaml`

**Comment by mmascher** [comment]
Hi @brianhlin ,

I have done the changes. Also, can you check if these info in 99-local.ini and in the factory are correct:

**99-local.ini**:
```
[Site Information]
...
resource = ELSA
resource_group = TCNJ
...
[Subcluster ELSA-TCNJ]
name = ELSA-TCNJ
```

**entry xml**:
```
<entry name="OSG_US_TCNJ-ELSA"...
...
            <attr name="GLIDEIN_ResourceName" const="True" glidein_publish="True" job_publish="True" parameter="True" publish="True" type="string" value="TCNJ-ELSA"/>
            <attr name="GLIDEIN_Site" const="True" glidein_publish="True" job_publish="True" parameter="True" publish="True" type="string" value="OSG_US_TCNJ-ELSA"/>
...
```

BTW, I changed the id because next_site_id is returning a new ID. Not sure if it is considering or not my previous branch..

**Comment by mmascher** [comment]
Hi @brianhlin ,

I have done the changes. Also, can you check if these info in 99-local.ini and in the factory are correct:

**99-local.ini**:
```
[Site Information]
...
resource = ELSA
resource_group = TCNJ
...
[Subcluster ELSA-TCNJ]
name = ELSA-TCNJ
```

**entry xml**:
```
<entry name="OSG_US_TCNJ-ELSA"...
...
            <attr name="GLIDEIN_ResourceName" const="True" glidein_publish="True" job_publish="True" parameter="True" publish="True" type="string" value="TCNJ-ELSA"/>
            <attr name="GLIDEIN_Site" const="True" glidein_publish="True" job_publish="True" parameter="True" publish="True" type="string" value="OSG_US_TCNJ-ELSA"/>
...
```

BTW, I changed the id because next_site_id is returning a new ID. Not sure if it is considering or not my previous branch..

File: topology/The College of New Jersey/ELSA/ELSA.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Disable: false
+Production: true
+SupportCenter: Self Supported
+GroupID:
```
Comment (brianhlin) [inline topology/The College of New Jersey/ELSA/ELSA.yaml]:
```suggestion
GroupID: 518
```

File: topology/The College of New Jersey/ELSA/ELSA.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+Disable: false
+Production: true
+SupportCenter: Self Supported
+GroupID: 
+GroupDescription: The TCNJ-ELSA cluster
+Resources:
+  TCNJ-ELSA:
+    ID: 1032
```
Comment (brianhlin) [inline topology/The College of New Jersey/ELSA/ELSA.yaml]:
```suggestion
    ID: 1031
```

File: topology/The College of New Jersey/ELSA/SITE.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+LongName: The College of New Jersey
+Description: The The College of New Jersey TCNJ-ELSA cluster 
+
+ID: 10259
```
Comment (brianhlin) [inline topology/The College of New Jersey/ELSA/SITE.yaml]:
```suggestion
ID: 10258
```

File: topology/The College of New Jersey/FACILITY.yaml [hunk 4]
```diff
@@ -0,0 +1 @@
+ID: 10175
```
Comment (brianhlin) [inline topology/The College of New Jersey/FACILITY.yaml]:
```suggestion
ID: 10174
```

File: topology/The College of New Jersey/ELSA/ELSA.yaml [hunk 5]
```diff
@@ -0,0 +1,31 @@
+Disable: false
+Production: true
+SupportCenter: Self Supported
+GroupID: 
+GroupDescription: The TCNJ-ELSA cluster
+Resources:
+  TCNJ-ELSA:
```
Comment (brianhlin) [inline topology/The College of New Jersey/ELSA/ELSA.yaml]:
```suggestion
  OSG_US_NEWJERSEY_ELSA:
```

**Comment by mmascher** [comment]
@brianhlin , please see if the latest changes are ok to merge.

**Comment by brianhlin** [comment]
@mmascher I don't particularly care either way about the resource group name since it's not used for GRACC dashboards or anything like that. What matters in the RG case is that it's consistently set to the same value in the CE config (it's not even used by the factory, IIRC). So `ELSA.yaml` is fine and we could always rename it if we want.

**Pull request merged**